packages/reputation-miner/ReputationMiner.js

```diff
@@ -103,6 +103,11 @@ class ReputationMiner {
     this.nReputations = ethers.constants.Zero;
     this.reputations = {};
     this.gasPrice = ethers.utils.hexlify(20000000000);
+
+    // Are we using ganache? If so, note this so we add safety margin to gas estimates because ganache sucks at that
+    // See if we're talking to Ganache to fix a ganache crash (which, while fun to say, is not fun to see)
+    const clientString = await this.realProvider.send("web3_clientVersion");
+    this.ganacheClient = clientString.indexOf('TestRPC') !== -1;
```

Comment:
Can this be `this.isGanacheClient` instead? `ganacheClient` suggests that the type of the variable is a client, not a boolean.